web/html/src/components/ace-editor.tsx

```diff
@@ -7,25 +6,15 @@ type Props = {
   minLines: number;
   maxLines: number;
   readOnly: boolean;
-  onChange: (...args: any[]) => any;
+  onChange?: (...args: any[]) => any;
   className: string;
   id: string;
   content: React.ReactNode;
+  /** Obsolete, do not use */
+  name?: string;
```

Comment:
Looks like the only usage of the ace editor is in the `config-channels.tsx`: we pass the `name`, but we don't use it in the component. Can we remove it?

Reply:
Good shout, thanks for pointing this out. Might even be worth keeping an eye out in the future, perhaps we can cut this dependency out altogether at some point.